src/Kmd.Logic.Gateway.Automation/PreValidation/ApisPreValidation.cs

```diff
@@ -37,12 +37,21 @@ public IEnumerable<GatewayAutomationResult> ValidateAsync(PublishFileModel publi
 
                 foreach (var api in publishFileModel.Apis)
                 {
+                    var apiPrefix = $"API: {api.Name}, {api.Path}";
+
                     if (api.ApiVersions == null)
                     {
                         this.ValidationResults.Add(new GatewayAutomationResult { IsError = true, ResultCode = ResultCode.ValidationFailed, Message = $"Version shouldn't be null" });
                         continue;
                     }
 
+                    if (api.ApiVersions
+                            .Where(v => v.IsCurrent.HasValue)
```

Comment:
We could get validation error due to this check, we need this check but need to rethink on how yaml looks.

Reply:
What do you mean by that?

Reply:
We have two Iscurrent set to true, in the yaml. If that is fixed this makes sense.